setup.cfg

```diff
@@ -15,10 +15,6 @@ show_source=true
 count=true
 
 [isort]
-include_trailing_comma=true
+profile=black
 known_first_party=dvc,tests
-known_third_party=PyInstaller,RangeHTTPServer,boto3,colorama,configobj,distro,dpath,flaky,flufl,funcy,git,grandalf,mock,moto,nanotime,networkx,packaging,pathspec,pygtrie,pylint,pytest,requests,ruamel,setuptools,shortuuid,shtab,toml,tqdm,voluptuous,yaml,zc
 line_length=79
```

Comment:
@efiop @pared @pmrowla, what do you guys say of increasing line length limit to black's default (i.e. 88)?

Reply:
Personally, I think 79 is fine. I'm not opposed to using 88, but I would prefer we don't go much higher than that in the future.

IMO Python projects that use 100+ tend to get hard to read, but I get that this is more of a personal taste issue than anything else.

Reply:
I have no strong opinion on that, only thing that bugs me is that this decision will require some refactor, as black tends to not remove `"` when its not needed anymore, so we will probably have a lot of lines like
`"This is " "the comment"`

Reply:
@pared, string handling has been significantly improved on black (https://github.com/psf/black/pull/1132) and should solve that issue. Though that'd be useful only if there were a new release. :(

Reply:
Okay then, if no one has *strong* opinion, I'll be increasing the limit to 88.

Reply:
Oops, missed this one. I would not increase it to 88. 80 is a good standard width that is used in many projects and is on the border of being revieable in github split screen view for PRs. Plus, if we change it now, we'll introduce inconsistency with the rest of the code base. So I'm not really seeing good reasons to go to 88, it won't make it easier to navigate.